src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -24,11 +24,9 @@ import { ToolsGridContainer } from '../styledComponents';
 import Contact, { PhoneNumber, PhoneNumberData } from './Contact';
 import {
   PersonInvalidNumberFragment,
-  PersonPhoneNumberFragment,
   useGetInvalidPhoneNumbersQuery,
 } from './GetInvalidPhoneNumbers.generated';
 import { useUpdateInvalidPhoneNumbersMutation } from './UpdateInvalidPhoneNumbers.generated';
-import { determineBulkDataToSend } from './helper';
```

Comment:
Since we no longer need `determineBulkDataToSend`, we can go ahead and delete the `./helper.ts` file.

This made me wonder, since we pass down a state value which triggers the submit call. What happens if you have 100 fixPhoneNumbers, will this fire all 100? Will all 100 fire if they are not all rendered on the page?